numpy/lib/tests/test_histograms.py

```diff
@@ -346,6 +346,19 @@ def test_precision(self):
         self.do_precision(np.single, np.longdouble)
         self.do_precision(np.double, np.longdouble)
 
+    def test_histogram_bin_edges(self):
+        hist, e = histogram([1, 2, 3, 4], [1, 2])
+        edges = histogram_bin_edges([1, 2, 3, 4], [1, 2])
+        assert_array_equal(edges, e)
+
+        arr = np.array([0.,  0.,  0.,  1.,  2.,  3.,  3.,  4.,  5.])
+        hist, e = histogram(arr, bins=30, range=(-0.5, 5))
+        edges = histogram_bin_edges(arr, bins=30, range=(-0.5, 5))
+        assert_array_equal(edges, e)
+
+        hist, e = histogram(arr, bins='auto', range=(0, 1))
+        edges = histogram_bin_edges(arr, bins='auto', range=(0, 1))
+        assert_array_equal(edges, e)
 
```

Comment:
nit: PEP8 wants two newlines between one class and the next - so one more here

Reply:
Done.